optuna_dashboard/ts/components/GraphIntermediateValues.tsx

```diff
@@ -1,102 +1,22 @@
-import { Box, Card, CardContent, Typography, useTheme } from "@mui/material"
-import * as plotly from "plotly.js-dist-min"
-import React, { FC, useEffect } from "react"
+import { Card, CardContent } from "@mui/material"
+import { PlotIntermediateValues } from "@optuna/react"
+import React, { FC } from "react"
 import { Trial } from "ts/types/optuna"
-import { usePlotlyColorTheme } from "../state"
-
-const plotDomId = "graph-intermediate-values"
 
 export const GraphIntermediateValues: FC<{
   trials: Trial[]
   includePruned: boolean
   logScale: boolean
 }> = ({ trials, includePruned, logScale }) => {
-  const theme = useTheme()
-  const colorTheme = usePlotlyColorTheme(theme.palette.mode)
-
-  useEffect(() => {
-    plotIntermediateValue(trials, colorTheme, false, !includePruned, logScale)
-  }, [trials, colorTheme, includePruned, logScale])
-
   return (
     <Card>
       <CardContent>
-        <Typography
-          variant="h6"
-          sx={{ margin: "1em 0", fontWeight: theme.typography.fontWeightBold }}
-        >
-          Intermediate values
-        </Typography>
-        <Box component="div" id={plotDomId} sx={{ height: "450px" }} />
+        <PlotIntermediateValues
+          trials={trials}
+          includePruned={includePruned}
+          logScale={logScale}
+        />
       </CardContent>
     </Card>
   )
 }
-
-const plotIntermediateValue = (
-  trials: Trial[],
-  colorTheme: Partial<Plotly.Template>,
-  filterCompleteTrial: boolean,
-  filterPrunedTrial: boolean,
-  logScale: boolean
-) => {
-  if (document.getElementById(plotDomId) === null) {
-    return
-  }
-
-  const layout: Partial<plotly.Layout> = {
-    margin: {
-      l: 50,
-      t: 0,
-      r: 50,
-      b: 0,
-    },
-    yaxis: {
-      title: "Objective Value",
-      type: logScale ? "log" : "linear",
-    },
-    xaxis: {
-      title: "Step",
-      type: "linear",
-    },
-    uirevision: "true",
-    template: colorTheme,
-    legend: {
-      x: 1.0,
-      y: 0.95,
-    },
```

Comment:
Could you add this property on `tslib` side?

Reply:
Added